### PR TITLE
Update READMEs

### PR DESCRIPTION
**Pull request by benmccann**
At first I was just going to remove this thinking that there was no need for it given the README in the main repo, but then I decided it'd be useful to have some minimal content here to help users who might want to contribute and are navigating the directory structure and also realized that it probably shows up on npm

### CODE REVIEW

**Comment by Rich-Harris** [comment]
Given that it _does_ show up on npm (https://www.npmjs.com/package/@sveltejs/kit) I think it should probably be more fleshed out. Calling it a 'library' is probably a bit of a misnomer; at its heart it's a CLI. The only time you'd actually import `@sveltejs/kit` that I can think of is when you need to import types

**Comment by benmccann** [comment]
I updated the description a bit based on your feedback. I also added a link to the work-in-progress docs. I'd hope not to duplicate too much from them and just be able to point people there

File: packages/kit/README.md [hunk 1]
```diff
@@ -1,15 +1,7 @@
...
-It's currently missing a ton of stuff but I figured I'd throw it up on GitHub anyway partly for the sake of 'working in the open' but mostly because I need an issue tracker to organise my thoughts.
-
-There are no tests yet or anything like that. Some of the code has just been straight copied over from the existing Sapper repo, but a pleasing amount of it can safely be left behind.
-
-## Trying it out
-
-Clone this repo, `npm install`, and `npm link`. That will create a global link to the `svelte` bin. You can then either `npm run build` or `npm run dev`, if you intend to make changes and see them immediately reflected.
-
-Then, clone the corresponding [svelte-app-demo](https://github.com/sveltejs/svelte-app-demo) repo and follow the instructions therein.
+Please see [the documentation](https://kit.svelte.dev/docs) for information about getting started and developing with SvelteKit.
```
Comment (Conduitry) [inline packages/kit/README.md:7]: Is there a particular reason to point to this page rather than just to the root of the Kit site as a whole?

Comment (benmccann) [reply packages/kit/README.md:7]: I debated that. It seemed like it'd save most users a click in getting to the docs. The landing page is just marketing and we don't really need users to see it first on their way to the docs

File: packages/kit/README.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
-# @sveltejs/kit
+# The fastest way to build Svelte apps
 
-Here be dragons, etc etc.
+This is the [SvelteKit](https://kit.svelte.dev) framework and CLI.
```
Comment (Conduitry) [inline packages/kit/README.md:3]: The trailing space is making linting fail.

If you're using VS Code and you have the appropriate extensions installed, then you _should_ already be getting format-on-save happening, and this sort of thing shouldn't be a problem.

**Comment by hyunbinseo** [comment]
[npm](https://www.npmjs.com/package/@sveltejs/kit) is showing the older version of `packages/kit/README.md`.

As a result, link to [change-log](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md) is not accessible in npm.

`Repository` and `Homepage` links should be set as well.

image

**Comment by Conduitry** [comment]
Yes. That will resolve itself once we start publishing as `@latest` on npm rather than just `@next`.

**Pull request merged**